commit efd3fb018c2b6d9aacc367ab29dc4b0089ba10a5
Author: Paul Voss <vosspa@rs211412.rs.hosteurope.de>
Date:   Fri Mar 14 10:43:30 2014 +0100

    - fixed syntax error by added missing closing bracket as mentioned in issue #44

Index: pkg-nagios-plugins-contrib/check_email_delivery/src/check_imap_quota
===================================================================
--- pkg-nagios-plugins-contrib.orig/check_email_delivery/src/check_imap_quota
+++ pkg-nagios-plugins-contrib/check_email_delivery/src/check_imap_quota
@@ -153,7 +153,7 @@ eval {
 	  }
 	  if (!length($quotaUsed) && !length($quotaLimit)) {
 		print "no answer from imap host\n" if $verbose > 2;
-	  } elsif (!length($quotaUsed) || !length($quotaLimit) {
+	  } elsif (!length($quotaUsed) || !length($quotaLimit)) {
 		print "incorrect answer from imap host\n";
 	$imap->close();
 	exit $status{UNKNOWN};
